aws/resource_aws_apigatewayv2_integration_test.go

```diff
@@ -445,17 +563,63 @@ resource "aws_apigatewayv2_integration" "test" {
 func testAccAWSAPIGatewayV2IntegrationConfig_httpProxy(rName string) string {
 	return testAccAWSAPIGatewayV2IntegrationConfig_apiHttp(rName) + fmt.Sprintf(`
 resource "aws_apigatewayv2_integration" "test" {
-	api_id           = "${aws_apigatewayv2_api.test.id}"
-	integration_type = "HTTP_PROXY"
+  api_id           = "${aws_apigatewayv2_api.test.id}"
+  integration_type = "HTTP_PROXY"
 
-	integration_method = "GET"
-	integration_uri    = "https://example.com"
+  integration_method = "GET"
+  integration_uri    = "https://example.com"
 }
 `)
 }
 
-func testAccAWSAPIGatewayV2IntegrationConfig_vpcLink(rName string) string {
-	return testAccAWSAPIGatewayV2IntegrationConfig_apiWebSocket(rName) + fmt.Sprintf(`
+func testAccAWSAPIGatewayV2IntegrationConfig_vpcLinkHttp(rName string) string {
+	return composeConfig(
+		testAccAWSAPIGatewayV2IntegrationConfig_vpcLinkHttpBase(rName),
+		fmt.Sprintf(`
+resource "aws_apigatewayv2_integration" "test" {
+  api_id           = "${aws_apigatewayv2_api.test.id}"
+  integration_type = "HTTP_PROXY"
+
+  connection_type      = "VPC_LINK"
+  connection_id        = "${aws_apigatewayv2_vpc_link.test.id}"
+  description          = "Test private integration"
+  integration_method   = "GET"
+  integration_uri      = "${aws_lb_listener.test.arn}"
+  timeout_milliseconds = 5001
+
+  tls_config {
+    server_name_to_verify = "www.example.com"
+  }
+}
+`))
+}
+
+func testAccAWSAPIGatewayV2IntegrationConfig_vpcLinkHttpUpdated(rName string) string {
+	return composeConfig(
+		testAccAWSAPIGatewayV2IntegrationConfig_vpcLinkHttpBase(rName),
+		fmt.Sprintf(`
+resource "aws_apigatewayv2_integration" "test" {
+  api_id           = "${aws_apigatewayv2_api.test.id}"
+  integration_type = "HTTP_PROXY"
+
+  connection_type      = "VPC_LINK"
+  connection_id        = "${aws_apigatewayv2_vpc_link.test.id}"
+  description          = "Test private integration updated"
+  integration_method   = "POST"
+  integration_uri      = "${aws_lb_listener.test.arn}"
+  timeout_milliseconds = 4999
+
+  tls_config {
+    server_name_to_verify = "www.example.org"
```

Comment:
no update to value from `testAccAWSAPIGatewayV2IntegrationConfig_vpcLinkHttp`?

Reply:
Value in `testAccAWSAPIGatewayV2IntegrationConfig_vpcLinkHttp` is _www.example._***com***.

Reply:
Doh, missed that